lib/matplotlib/category.py

```diff
@@ -80,22 +61,7 @@ def __init__(self, seq):
 
 
 def convert_to_string(value):
```

Comment:
you should be able to get rid of this function entirely as all the data gets cast to strings with the np.array(dtype=unicode) call and therefore you can remove the call to convert_to_string. Also, you should then remove the test functions from category.py